README.md

```diff
@@ -17,13 +17,11 @@ Then, using [pip](https://pip.pypa.io/en/stable/installing/) package manager for
 ```
 sudo pip install bottle
 sudo pip install gevent-websocket
-sudo pip install pymongo  # optional
+sudo pip install pymongo
```

Comment:
Is required even if you don't want save sessions on a database

Reply:
nope.. o no hauria de serho